llama/src/main/java/com/cloudera/llama/am/api/LlamaAM.java

```diff
@@ -95,7 +95,7 @@
   public static final long GANG_ANTI_DEADLOCK_BACKOFF_MAX_DELAY_DEFAULT = 30000;
 
   public static final String CACHING_ENABLED_KEY =
-      PREFIX_KEY + "caching.enabled";
+      PREFIX_KEY + "cache.enabled";
```

Comment:
Thanks for updating this. Can we also update the reference to "caching.enabled" in llama-site.xml as well?